references/video_classification/presets.py

```diff
@@ -31,7 +30,6 @@ class VideoClassificationPresetEval:
     def __init__(self, *, crop_size, resize_size, mean=(0.43216, 0.394666, 0.37645), std=(0.22803, 0.22145, 0.216989)):
         self.transforms = transforms.Compose(
             [
-                ConvertBHWCtoBCHW(),
```

Comment:
Why can we simply remove this? It seems something is off here. The old `Kinetics400` states

https://github.com/pytorch/vision/blob/970ba3555794d163daca0ab95240d21e3035c304/torchvision/datasets/kinetics.py#L292

whereas the new `Kinetics` states

https://github.com/pytorch/vision/blob/970ba3555794d163daca0ab95240d21e3035c304/torchvision/datasets/kinetics.py#L69

Was this intentional or did we botch this during the introduction of the new class? In any way, we need to adapt our deprecation warnings accordingly:

https://github.com/pytorch/vision/blob/970ba3555794d163daca0ab95240d21e3035c304/torchvision/datasets/kinetics.py#L252-L254

https://github.com/pytorch/vision/blob/970ba3555794d163daca0ab95240d21e3035c304/torchvision/datasets/kinetics.py#L308-L311